src/lib/Input.svelte

```diff
@@ -24,8 +34,37 @@
         messageReceived = false;
     };
 
+    /**
+     * Returns a boolean indicating whether the innerText contains letters.
+     *
+     * @returns {boolean} Whether the innerText contains letters.
+     */
+    function innerTextContainsLetters(): boolean {
+        return innerText.match(/[a-z]/i) !== null;
+    }
+
     let canSend = false;
-    $: canSend = Boolean(innerText) && innerText.length > 0 && messageReceived;
+    $: canSend =
+        Boolean(innerText) &&
+        innerText.length > 0 &&
+        messageReceived &&
+        innerTextContainsLetters();
+
+    function handleKeyDown(event: {
```

Comment:
Here's a PR to change up this file a little bit: https://github.com/mitmedialab/chat-tutor-embeddable/pull/8

The changes are:
- use `KeyboardEvent` for argument type (nice, built-in type)
- invert if-statement so there's less nesting (which I think is easier to follow / get right, let me know if you agree/disagree)
- The logic if the `else if` branch is redundant / a little incorrect (since the outer if statement already checked `event.key === "Enter" && !event.shiftKey`)

Reply:
If you like the look of those changes, you can merge the PR